include/DDpackage.h

```diff
@@ -71,6 +71,20 @@ namespace dd {
         ListElementPtr next;
         int            w;
     };
+    struct Control {
+        enum controlType : bool { pos = true,
+                                  neg = false };
+
+        unsigned short qubit = 0;
+        controlType    type  = pos;
+
+        explicit Control(unsigned short qubit = 0, controlType type = pos):
```

Comment:
Does a default arugment for the qubit make sense?

Reply:
This could be necessary to default construct a `std::vector<Contol>`.